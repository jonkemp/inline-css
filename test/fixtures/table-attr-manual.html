<html>
<head>
<style>
    .table {
        background-color: red;
        float: left;
        width: 100%;
        height: 500px;
        padding: 0;
        border: 0;
        border-spacing:0;
    }
    .tr {
        background-color: blue;
        vertical-align: top;
        text-align: left;
    }
    .th, .td {
        background-color: green;
        width: 50%;
        height: 250px;
        vertical-align: bottom;
        text-align: right;
        white-space: nowrap;
    }
    .thead, .tbody, .tfoot {
        vertical-align: baseline;
        text-align: center;
    }
</style>
</head>
<body>
    <table>
        <tr>
            <td>
                <table border="0">
                    <tr>
                        <td>
                            empty
                        </td>
                    </tr>
                </table>
            </td>
        </tr>
    </table>
    <table border="1" cellpadding="2" cellspacing="3">
        <tr>
            <td>
                empty
            </td>
        </tr>
    </table>
    <table class="table">
        <thead class="thead">
            <tr>
                <th class="th">
                    th
                </th>
            </tr>
        </thead>
        <tbody class="tbody">
            <tr class="tr">
                <td class="td">
                    td
                </td>
            </tr>
        </tbody>
        <tfoot class="tfoot">
            <tr>
                <td>
                    td
                </td>
            </tr>
        </tfoot>
    </table>
</body>
</html>
