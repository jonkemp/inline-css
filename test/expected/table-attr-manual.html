<html>
<head>

</head>
<body>
    <table>
        <tr>
            <td>
                <table border="0">
                    <tr>
                        <td>
                            empty
                        </td>
                    </tr>
                </table>
            </td>
        </tr>
    </table>
    <table border="1" cellpadding="2" cellspacing="3">
        <tr>
            <td>
                empty
            </td>
        </tr>
    </table>
    <table class="table" style="" align="left" bgcolor="red" width="100%" height="500px" cellpadding="0" border="0" cellspacing="0">
        <thead class="thead" style="" valign="baseline" align="center">
            <tr>
                <th class="th" style="" bgcolor="green" width="50%" height="250px" valign="bottom" align="right" nowrap="nowrap">
                    th
                </th>
            </tr>
        </thead>
        <tbody class="tbody" style="" valign="baseline" align="center">
            <tr class="tr" style="" bgcolor="blue" valign="top" align="left">
                <td class="td" style="" bgcolor="green" width="50%" height="250px" valign="bottom" align="right" nowrap="nowrap">
                    td
                </td>
            </tr>
        </tbody>
        <tfoot class="tfoot" style="" valign="baseline" align="center">
            <tr>
                <td>
                    td
                </td>
            </tr>
        </tfoot>
    </table>
</body>
</html>
